LICENSE.md

```diff
@@ -0,0 +1,64 @@
+This Software License Agreement (“Agreement”) defines the terms under which the Software is licensed. This Agreement is a legally binding contract that controls and limits your rights to use SAIC’s Software. Your use of the Software indicates your acceptance of this Agreement. Do not use the Software if you disagree with any provision of this Agreement.
+
+DEFINITIONS
+
+“Licensor” means Science Applications International Corporation (also, “SAIC”, “we”, “our”, “us”).
+“Licensee” means the individual, company, or other legal entity utilizing the Software (also “you”).
+“Software” means the Maryland Test Facility Public Code and Data Repository.
+“Government Entity” means any federal, state, provincial, or local government, agency, department, division or instrumentality thereof.
+
+TERMS AND CONDITIONS
+
+1.	License Grant.  Licensor grants to Licensee a non-exclusive, non-transferable, non-sublicensable, limited license (as set forth herein) to use the Software, subject to all terms and conditions of this Agreement.   The Software is licensed for use (1) by Government Entities for research, development, and evaluation activities; and (2) for use by commercial biometric and identity product manufacturers for the sole purpose of integrating their technologies with the Software/API to support Government research, development, and evaluation activities. Any other use is strictly prohibited. This prohibition includes, but is not limited to, the use of the Software to provide technology testing services.  Any rights not granted to you are exclusively reserved by us. 
+
+2.	Fees.  Subject to the terms and conditions of this Agreement, you are not obligated to pay any fees to SAIC for the license to the Software granted to you under this Agreement.
+
+3.	Restrictions.
+a.	Licensee shall not modify, translate, or adapt in any way, or otherwise create derivative works, modifications or improvements, whether or not patentable, of the Software or Documentation or any part thereof, and/or use the Software or Documentation for purposes of competitive analysis of the Software, the development of a competing software product or service, or any other purpose that is to the Licensor’s commercial disadvantage;
+b.	Licensee shall not reverse engineer, disassemble, decompile, decode, or otherwise attempt to derive or gain access to the source code of the Software or any part thereof;
```

Comment:
Suggest revisions, by cloning the repository the licensee is gaining access to the source code.